### PR TITLE
fix tabobject

### PR DESCRIPTION
**Pull request by PingHsunTsai**
### What type of change is this?

- [ ] Bug fix in a `tabobject`

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -39,7 +39,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 * Restructured `SceneObjects` to accepct `item` as kwargs.
 * Fixed `compas_viewer.components.slider` step attribute.
 * Fixed `renderer.view` distorted init
-
+* Fixed in a `tabobject`
```
Comment (tomvanmele) [inline CHANGELOG.md:42]: please leave a blank line

Comment (tomvanmele) [reply CHANGELOG.md:42]: and put a period at the end of every sentence

File: src/compas_viewer/scene/tagobject.py [hunk 2]
```diff
@@ -118,6 +118,7 @@ def make_buffers(self):
         }
 
     def make_text_texture(self):
+        print(self.geometry.font)
```
Comment (tomvanmele) [inline src/compas_viewer/scene/tagobject.py:121]: ?

Comment (tomvanmele) [reply src/compas_viewer/scene/tagobject.py:121]: is this the hotfix?

Comment (PingHsunTsai) [reply src/compas_viewer/scene/tagobject.py:121]: Ahh no there was a missing file, I will remove this print line

File: CHANGELOG.md [hunk 3]
```diff
@@ -38,7 +38,8 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 * Changed `FrameObject` to something with just 3 axes.
 * Restructured `SceneObjects` to accepct `item` as kwargs.
 * Fixed `compas_viewer.components.slider` step attribute.
-* Fixed `renderer.view` distorted init
+* Fixed `renderer.view` distorted init.
+* Fixed in a `tabobject`.
```
Comment (tomvanmele) [inline CHANGELOG.md:42]: `tagobject`?

Comment (tomvanmele) [reply CHANGELOG.md:42]: fixed what in `tagobject`?

**Review by tomvanmele — COMMENTED** [review]
i still don't get it. where are all the fixes in this PR? i don't see any changes...

File: src/compas_viewer/configurations/default_config/FreeSans.ttf
Comment (PingHsunTsai) [inline src/compas_viewer/configurations/default_config/FreeSans.ttf:1]: By putting back the missing file here, tagobject file can be fixed

File: CHANGELOG.md (hunk 3 above)
Comment (PingHsunTsai) [reply CHANGELOG.md:42]: sry didn't see the typo

**Review by tomvanmele — APPROVED** [review]

**Pull request merged**